rs/tests/message_routing/xnet/xnet_compatibility.rs

```diff
@@ -146,7 +146,11 @@ pub async fn test_async(env: TestEnv) {
         .map(|(_, _, node)| node)
         .map(|node| runtime_from_url(node.get_public_url(), node.effective_canister_id()));
 
-    let xnet_config = xnet_slo_test_lib::Config::new(2, 1, Duration::from_secs(30), 10);
+    // Only guaranteed response calls, for now.
+    // TODO(MR-638): Also enable best-effort calls, when supported on mainnet.
+    let call_timeouts = [None];
```

Comment:
What is the thinking to not use best-effort responses here while using them in the other tests?

Reply:
Setting a call timeout on the mainnet replica (which this test will do, as it tries upgraing from and then downgrading to the mainnet version) will panic. We can wait with merging this until the mainnet replica no longer panics when setting a call timeout; or run the test with guaranteed response calls only for the time being.

Reply:
Ah, I forgot about this case. Makes total sense to merge as is.